### DISCUSSION TITLE
Sort notable contributions by number of stars ?

### DISCUSSION THREAD

**Discussion by bokub** [lowlighter/metrics#1021] [Ideas]
I just realized that my 1 year old PR #293 cannot work (and never actually worked) because of the `sort` at the end of this line :

https://github.com/lowlighter/metrics/blob/ead26f7a0cae42cf5796515ff05300e58f84ba6a/source/plugins/notable/index.mjs#L31-L32

Is it intended to sort alphabetically ?

![image](https://user-images.githubusercontent.com/17952318/165384524-d93725de-ff9b-443a-a6b5-182e34d3efcb.png)

If not, I can make the PR to fix this and sort by number of ⭐ , as planned originally in my PR.

Thanks for your answer

**Comment by lowlighter** [comment]
Hi!

I think the alphabetical sort was there just to avoid the random ordering between runs, so it's no longer useful indeed

You can open a PR to remove this 👍 
Thanks a lot for this follow-up
